app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,25 @@
+class MessagesController < ApplicationController
+  before_action :authenticate_user!
+  respond_to :json
+
+  def create
+    @message = Message.new(message_params)
+
+    if @message.save
+      broadcast_message(@message)
```

Comment:
You don't need to pass `@message` to `broadcast_message ` method given `@message` is a global variable.
Just use it.